packages/neo-one-client/src/provider/neoone/JSONRPCClient.js

```diff
@@ -210,4 +210,10 @@ export default class JSONRPCClient {
       method: 'getnetworksettings',
     });
   }
+
+  startConsensusNow(): Promise<void> {
```

Comment:
nit: probably call this and all of the methods `runConsensusNow` to be consistent?